content/blog/tech-preview-oct-2019.md

```diff
@@ -0,0 +1,40 @@
++++
+categories = ["Community", "Newsletter"]
+date = "2019-10-30"
+tags = ["nats", "microservices", "technical", "features", "jetstream"]
+title = "Sneak Peek Tech Review"
+author = "Derek Collison"
++++
+
+Our team is always working to bring you features to improve your NATS experience. The following are features coming soon to a NATS Server near you!
```

Comment:
Most are already done. Only one not released is JetStream, we just need to formalize docs. Need to reword.

Reply:
Is there a release I can reference for the features released?

Reply:
Some are in 2.0.4 and some in 2.1.0, @kozlovic might be able to help, or look at release notes.

Reply:
Latency tracking in 2.1.0
Response Permissions in 2.0.4
Queue Permission coming up in next release IIRC.

Reply:
Next release will likely be 2.1.1 (since Derek wants to keep 2.2.0 for JS ;-)). That being said nothing is in stone yet.